### PR TITLE
pyquery: 1.2.4 -> 1.2.9

### PR DESCRIPTION
**Pull request by rnhmjoj**
Requires #10332

### CODE REVIEW

**Comment by peti** [comment]
Travis-CI complains about a syntax error: https://travis-ci.org/NixOS/nixpkgs/builds/84832096#L116.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -12188,11 +12188,12 @@ let
 
 
   pyquery = buildPythonPackage rec {
-    name = "pyquery-1.2.4";
+    name = "pyquery-${version}";
+    version = "1.2.9"
```
Comment (peti) [inline pkgs/top-level/python-packages.nix]: This line lacks a semicolon.

Comment (rnhmjoj) [inline pkgs/top-level/python-packages.nix]: Thanks, I messed up the gut history and I couldn't test it afterward.

**Comment by domenkozar** [comment]
`
curl: (22) The requested URL returned error: 404 Not Found
error: cannot download pyquery-1.2.9.tar.gz from any mirror
``

**Pull request merged**